helm-chart-sources/logstream-workergroup/README.md

```diff
@@ -41,6 +41,7 @@ This section covers the most likely values to override. To see the full scope of
 |config.probes|true|enables (true) or disables (false) the liveness and readiness probes.|
 |config.livenessProbe|see `values.yaml`|[livenessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-a-liveness-http-request) configuration|
 |config.readinessProbe|see `values.yaml`|[readinessProbe](https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-startup-probes/#define-readiness-probes) configuration|
+|criblImage.wolfiImage|false|Cribl publishes 2 versions of the container, the default based on Ubuntu and the alternative version based on Wolfi.  This setting should only be enabled if you want to use Cribls published Wolfi container image.|
```

Comment:
Nitpick: would you mind moving the new
```
cribllImage.wolfImage
```
property up so it sits next to
```
cribllImage.tag
```
Keeping cribllImage related fields (like “tag” and its associated “wolfImage”) grouped together makes it much easier to scan
Simillar comment applies to rest of READMe's